public/style.css

```diff
@@ -73,6 +73,22 @@ hr {margin-top: 1.7rem;}
 
 }
 
+.flag-wrapper {
+   text-align: center;
+   margin-top: -35px;
+   margin-bottom: 5px;
+}
+
+.flag-wrapper > img {
+   display: inline-block;
+   margin-left: 90px;
+}
+
+#flag-image {
+  margin-top: 5px;
+  width: 40px;
+}
+
```

Comment:
Can you simplify these? "#flag-image" and ".flag-wrapper > img" refer to the same element. You can also combine margin-top with .flag-wrapper - currently you're subtracting 35 then adding back 5. Could just subtract 30 on the wrapper and call it done. text-align and display properties don't appear to do anything here either.

Reply:
Oof this was cruft leftover from the old implementation.  You're right, it didn't need most of these